### PR TITLE
[Docs] updating information about fetch caching in the fetching docs

### LINKED ISSUE #72152: Docs: Updating the docs about fetch being memoized
### What is the documentation issue?

There is incorrect information about how fetch works in the new Next.js version (^15.0.0).
The documentation says that fetch is automatically memoized in the said version but now by default fetchs are no longer being memoized

### Is there any context that might help us understand?

[Link of the section](https://nextjs.org/docs/app/building-your-application/data-fetching/fetching#reusing-data-across-multiple-functions)

### Does the docs page already exist? Please link to it.

https://nextjs.org/docs/app/building-your-application/data-fetching/fetching#reusing-data-across-multiple-functions

### PR DESCRIPTION
**Pull request by RafaelAndeloci**
Closes: https://github.com/vercel/next.js/issues/72152

This PR fixes a information about fetching being memoized by default in the [fetching docs](https://nextjs.org/docs/app/building-your-application/data-fetching/fetching#reusing-data-across-multiple-functions)

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: b0354aa431c90c092247e4a8bf4c310ec64b9da3

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by RafaelAndeloci** [comment]
it's my first contribution, do I need to do something else in order to pass the checks?

File: docs/02-app/01-building-your-application/02-data-fetching/01-fetching.mdx [hunk 1]
```diff
@@ -263,7 +263,7 @@ This example caches the result of the database query for 1 hour (3600 seconds).
 
 Next.js uses APIs like `generateMetadata` and `generateStaticParams` where you will need to use the same data fetched in the `page`.
 
-If you are using `fetch`, requests are automatically [memoized](/docs/app/building-your-application/caching#request-memoization). This means you can safely call the same URL with the same options, and only one request will be made.
+If you are using `fetch`, requests are no longer being automatically [memoized](/docs/app/building-your-application/caching#request-memoization). Read the fetch changes [here](/docs/app/building-your-application/upgrading/version-15#fetch-requests)
```
Comment (ijjk) [inline docs/02-app/01-building-your-application/02-data-fetching/01-fetching.mdx:266]:
```suggestion
If you are using `fetch`, requests can be [memoized](/docs/app/building-your-application/caching#request-memoization) by adding `cache: 'force-cache'`. This means you can safely call the same URL with the same options, and only one request will be made.
```

Maybe we mention how to opt into the caching as it's the point of the example below.

Comment (RafaelAndeloci) [reply docs/02-app/01-building-your-application/02-data-fetching/01-fetching.mdx]: i'll add it to the code example too

**Comment by RafaelAndeloci** [comment]
@ijjk updated the code example and put a good to know card explaining that the default `cache: no-store` changed in version 15.

File: docs/02-app/01-building-your-application/02-data-fetching/01-fetching.mdx [hunk 2]
```diff
@@ -263,7 +263,11 @@ This example caches the result of the database query for 1 hour (3600 seconds).
 
 Next.js uses APIs like `generateMetadata` and `generateStaticParams` where you will need to use the same data fetched in the `page`.
 
-If you are using `fetch`, requests are automatically [memoized](/docs/app/building-your-application/caching#request-memoization). This means you can safely call the same URL with the same options, and only one request will be made.
+If you are using `fetch`, requests can be [memoized](/docs/app/building-your-application/caching#request-memoization) by adding `cache: 'force-cache'`. This means you can safely call the same URL with the same options, and only one request will be made.
+
+> **Good to know:**
+>
+> - In previous versions of Next.js, using `fetch` will automatically sets the `cache` to `cache: force-cache`. This changed in version 15, to `cache: no-store`
```
Comment (ijjk) [inline docs/02-app/01-building-your-application/02-data-fetching/01-fetching.mdx:270]:
```suggestion
> - In previous versions of Next.js, using `fetch` would have a default `cache` value of `force-cache`. This changed in version 15, to a default of `cache: no-store`.
```

**Review by ijjk — APPROVED** [review]

**Comment by RafaelAndeloci** [comment]
@ijjk thanks bro!

**Pull request merged**